cmd/virt-chroot/main_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Virt chroot sanity", func() {
+	It("virt-chroot should not be bloated with dependencies", func() {
+		path, err := os.Executable()
+		Expect(err).ToNot(HaveOccurred())
+		f, err := os.Stat(path)
+		Expect(err).ToNot(HaveOccurred())
+		limitMi := 10
```

Comment:
Where does this limit come from?

Reply:
It's a somewhat sane boundary considering the current virt-chroot compiled binary size